data/getting-started/pachyderm.md

```diff
@@ -21,19 +21,118 @@ Read about how our Customers leveraged Pachyderm to face their ML/AI challenges
+        ``` yaml
+            {
+                "Version": "2012-10-17",
+                "Statement": [
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:ListBucket"
+                        ],
+                        "Resource": [
+                            "<arn:aws:...>"
+                        ]
+                    },
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "s3:PutObject",
+                            "s3:GetObject",
+                            "s3:DeleteObject"
+                        ],
+                        "Resource": [
+                            "<arn:aws:...>/*"
+                        ]
+                    }
+                ]
+            }
+        ```
+
+
+
+- In Red Hat OpenShift Console, we will guide you through the creation of a new namespace for your cluster as well as a secret holding your Bucket's credentials.
+
+    * **A- Create a Project**
+    On the left menu, select Operator > OperatorHub.
+    On the top of the page, click on the Project Dropdown.
+    Choose a project name, then click Create.
 
-### Install the Operator
+    * **B- Create the Namespace in which you will deploy Pachyderm**        
+    On the left menu, select Administration > Namespaces, then create a new namespace for your cluster.
+
+    * **C- Create the Secret that will hold your user credentials and bucket name**, granting your cluster access to your bucket.
+  
+         Select Workloads > Secrets > Create a secret from YAML on the left menu.
+         
+         Choose your secret name, then fill in:
+
+         - your namespace
+         - your IAM user access key and IAM user secret
+         - your bucket's name
+         - your bucket's region.
+
+         **Attention Ceph Users!** You need to add an endpoint to the stringData section of your secret: `custom-endpoint:"http://ClusterIP:80"`
+	
+		 ```yaml 
+		 apiVersion: v1
+		 kind: Secret
+		 metadata:
+		    name: <name-your-secret>
+		    namespace: <your-namespace>
+		 type: Opaque
+		 stringData:
+			  access-id: <IAM-user-access-key>
+		    access-secret: <IAM-user-secret>
+		    bucket: <bucket-name>
+		    region: <bucket-region>
+		 ```
+		 For more information on creating a namespace, see [Red Hat Marketplace Docs](https://marketplace.redhat.com/en-us/documentation/clusters).
+
+## Install The Operator
 Pachyderm Operator has a **Red Hat marketplace listing**.
 
 - [Subscribe to the operator on Marketplace.](https://marketplace.redhat.com/en-us/products/pachyderm)
   
 - [Install the operator and validate.](https://marketplace.redhat.com/en-us/documentation/operators)
 
-### Post Deployment
+## Deploy Pachyderm
+
+Make sure to select the project you created above on the top of the screen. 
+
+On the Operator:
```

Comment:
I think it would be good to be more specific and guide the user to `Operators > Installed Operators` menu like done above while creating a new project/ns